Formula/sha1dc.rb

```diff
@@ -0,0 +1,36 @@
+class Sha1dc < Formula
+  desc "Tool to detect SHA-1 collisions in files, including SHAttered"
+  homepage "https://github.com/cr-marcstevens/sha1collisiondetection"
+
+  # The "master" branch is unusably broken and behind the "simplified_c90" branch
+  # that's the basis for release.
+  head "https://github.com/cr-marcstevens/sha1collisiondetection.git",
+       :using => :git,
+       :branch => "simplified_c90"
+
+  devel do
+    url "https://github.com/cr-marcstevens/sha1collisiondetection/archive/development-v1.0.1.tar.gz"
+    sha256 "a414aad7ab1da30193d053ad31651d39a449c2f9f74eac777e467929d0de3c93"
+    version "dev-v1.0.1"
```

Comment:
Having, queried, in https://github.com/cr-marcstevens/sha1collisiondetection/pull/19#issuecomment-284256955 an answer is offered:

> Version numbers (major.minor.patch) will be increasing and marked as stable or development. E.g. development-v1.0.1, and now stable-v1.0.2.